data/tribes/scripting/help/tips.lua

```diff
@@ -30,6 +30,11 @@ function get_frisian_tips()
    return tips
 end
 
+function get_addon_tribe_tips(tribe)
+   include "txts/tips/frisians.lua"
+   return tips
+end
+
```

Comment:
Where is this function used?

Reply:
leftover, will delete it

Reply:
You pointed me at `data/tribes/scripting/help/tips.lua`. The get_<tribe>_tips in there should be obsolete now too.